QUICK_START.md

```diff
@@ -0,0 +1,153 @@
+_MIT License_
+_Copyright (c) 2017 MassChallenge, Inc._
+
+## Quickstart for Impact-API
+
+1. Install [Docker](https://docs.docker.com/docker-for-mac/)
+(download the stable channel).
+
+2. Start Docker from the Applications folder.
+
+3. Get the source code.  If you already have impact-api checked out,
```

Comment:
I think it's worth clarifying what "Accelerate" is and that impact-api relies on it to manage its DB, and then clarify how it affects the installation.